client/gutenberg/extensions/tiled-gallery/edit.jsx

```diff
@@ -208,9 +208,7 @@ class TiledGalleryEdit extends Component {
 				{ controls }
 				<InspectorControls>
 					<PanelBody title={ __( 'Tiled gallery settings' ) }>
-						{ /* @TODO disable with title comment, don't remove */ layoutSupportsColumns(
```

Comment:
> disable with title comment, don't remove

Noting this comment by @MichaelArestad just for the record:

>  It would be cool to add a column slider in the block inspector that is only available when the columns view is active. Steal the design from the Columns block.

via https://github.com/Automattic/wp-calypso/pull/29760#issuecomment-450420103